From ebd6d6519c82ca14761199c48f0927a9c7a9fee9 Mon Sep 17 00:00:00 2001
From: Paul Cochrane <paul@liekut.de>
Date: Wed, 28 May 2014 22:07:47 +0200
Subject: [PATCH 1/2] Removing cygwin XS workaround in Makefile.PL

The issue mentioned in RT#75672 seems to have resolved itself via
updates to cygwin.
---
 Makefile.PL |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index 8731ae8..fc0ce04 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -14,11 +14,6 @@ foreach ( @ARGV ) {
 unless ( defined $make_xs ) {
 	$make_xs = can_xs();
 }
-if ( $^O eq 'cygwin' and $make_xs == 1 and not /^-xs/ ) {
-	# Cygwin goes bonkers breaking `` if using Params::Util XS version
-	# for no apparent reason.
-	$make_xs = 0;
-}
 
 # Generate the non-XS tests if we are making the XS version
 my @tests = qw{
-- 
1.7.9

